### ISSUE TITLE
Test: F6 and shift + F6 to navigate across workbench

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#96263]
Refs: https://github.com/microsoft/vscode/issues/94921

- [ ] anyOS @RMacfarlane
- [x] anyOS @eamodio

Complexity: 1

---

We have introduced "View: Focus Next Part" `F6` and "View: Focus Previous Part" `shift + F6` as commands to navigate across workbench.
The focus should go Editor > Status Bar > Side Bar > Panel. If one of them is not visible it should skip it. Verify
* Pressing F6 and Shift + F6 you can navigate across the workbench parts
* If one of the parts is not visible it just skips over it (for example panel)
* If focus is not in one of the parts, for example in the activity Bar both F6 and Shift + F6 navigate to the Editor area

**Comment by eamodio** [comment]
Looks good!
